### PR TITLE
facets: fixes search bug

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
Please, rebase with latest master.

**Comment by ioannistsanaktsidis** [comment]
For some reason the event that I have is not firing. I have to check the problem and I will amend the commit.

**Comment by ioannistsanaktsidis** [comment]
Rebased and amended the commit.

File: inspire/base/static/js/search/form.js [hunk 1]
```diff
@@ -32,6 +32,10 @@ define([
     return true; // ensure form still submits
   })
 
+  $("button[name=action_search]").on("click", function(){
```
Comment (jmartinm) [inline inspire/base/static/js/search/form.js]: When you add a search query to the search bar and press Enter, does it count as `click` ?

Comment (ioannistsanaktsidis) [inline inspire/base/static/js/search/form.js]: yes it counts as click.

2015-11-24 13:32 GMT+01:00 Javier Martin Montull <notifications@github.com>:

**Pull request merged**